website/static/js/fangorn.js

```diff
@@ -2230,6 +2259,23 @@ tbOptions = {
     uploads : true,         // Turns dropzone on/off.
     columnTitles : _fangornColumnTitles,
     resolveRows : _fangornResolveRows,
+    // Move obj.data.attributes contents to obj.data , for WB v1 API
+    lazyLoadPreprocess: function(obj) {
+        function moveAttrs(obj) {
```

Comment:
@felliott 
Minor:'s done.

I'm not sure I'm following on the Important: . I thought it was the same variable/object(obj) throughout. However, I made my best guess and certainly the renaming is easier to read. Please check if this is what you had in mind.